si-id-epu50/si_id_epu50/ecodrive.py

```diff
@@ -2,96 +2,87 @@
 
 import logging
 logger = logging.getLogger(__name__)
-import logging.handlers as handlers
 import socket
 import threading
 import time
 
 from . import constants as _cte
 from .utils import *
 
-logger.setLevel(logging.INFO)
-formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
-logHandler = handlers.RotatingFileHandler(filename='ecodrive.log', maxBytes=10*1024*1024)
-logHandler.setFormatter(formatter)
-logger.addHandler(logHandler)
-
-
 class EcoDrive():
 
     _lock = threading.RLock()
-    _SOCKET_TIMEOUT = 1 # tcp socket timeout
-
-    def __init__(self, address, max_limit=+25, min_limit=-25,
-                    bbb_hostname = None, rs458_tcp_port=None, drive_name = 'EcoDrive') -> None:
-        """."""
+    _TCP_SOCKET_TIMEOUT = 1 # ms
+
+    def __init__(self, socket: socket.socket, address: int, max_limit, min_limit,
+                 bbb_hostname, rs458_tcp_port: int = None, drive_name: str = 'EcoDrive') -> None:
+        
```

Comment:
There should no be an empty line before the method docstring. And it is necessary to include an empty line between the first docstring line and the other ones.